ios/FluentUI/Pill Button Bar/PillButtonBarTokens.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFPillButtonBarTokens: MSFTokensBase {
+    public var cornerRadius: CGFloat!
+    public var maxButtonsSpacing: CGFloat!
+    public var minButtonsSpacing: CGFloat!
+    public var minButtonVisibleWidth: CGFloat!
+    public var minButtonWidth: CGFloat!
+    public var minHeight: CGFloat!
+    public var sideInset: CGFloat!
+    public var hoverBackgroundColor: UIColor!
+    public var onBrandHoverBackgroundColor: UIColor!
+
+    override init() {
+        super.init()
+
+        self.themeAware = true
+
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
```

Comment:
How can I test to verify whether this is or isn't happening?

Reply:
Also note, CCB is UIKit, not SwiftUI, and does not have the themeDidUpdate/notification for updating tokens. So either we both have the same problem, or it isn't a problem.

Reply:
And one last thing - what is this coding paradigm called? I'd like to do a bit more research, but I'm not certain what to search for :)

Reply:
> Also note, CCB is UIKit, not SwiftUI, and does not have the themeDidUpdate/notification for updating tokens. So either we both have the same problem, or it isn't a problem.

FWIW, I'm fairly certain from my testing that CCB **does** have this same issue :) As they stand right now, it appears that our other UIKit vNext controls (at least the one's I've looked at) will only update to a new theme _after_ they are dismissed from their window and re-added. This isn't something that we would have noticed in our test app, since we always remove and re-add to change themes (by moving around the navigation stack).

@edjamesmsft, feel free to duplicate the logic used by the `DrawerController` and its `MSFDrawerTokens` that @rdeassis mentioned. That's also a UIKit control, and appears to be correct in its updating.

> And one last thing - what is this coding paradigm called? I'd like to do a bit more research, but I'm not certain what to search for :)

Not sure if it has a strict name, but it's basically a callback. Kinda like a delegation pattern-type thing, but not the "strict" `Delegate` pattern that UIKit often uses.

Reply:
https://office.visualstudio.com/APEX/_workitems/edit/5652140 for the task to fix this more correctly